common/lib/capa/capa/responsetypes.py

```diff
@@ -250,8 +251,15 @@ def render_html(self, renderer, response_msg=''):
           - renderer : procedure which produces HTML given an ElementTree
           - response_msg: a message displayed at the end of the Response
         """
-        # render ourself as a <span> + our content
-        tree = etree.Element('span')
+
+        # get responsetype index to make responsetype label
+        response_index = self.xml.attrib['id'].split('_')[-1]
+
+        # wrap the content inside a section
+        tree = etree.Element('section')
+        tree.set('class', 'wrapper-problem-response')
+        tree.set('tabindex', '-1')
+        tree.set('aria-label', RESPONSE_LABEL + ' ' + response_index)
```

Comment:
I am sorry @cahrens I didn't get you here. We are using a `aria-label` in the form of 'Question 1', 'Question 2' etc.